src/main/java/uk/gov/justice/digital/delius/jpa/standard/repository/AddressAssessmentDocumentRepository.java

```diff
@@ -1,10 +1,13 @@
 package uk.gov.justice.digital.delius.jpa.standard.repository;
 
 import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
 import uk.gov.justice.digital.delius.jpa.standard.entity.AddressAssessmentDocument;
 
 import java.util.List;
 
 public interface AddressAssessmentDocumentRepository extends JpaRepository<AddressAssessmentDocument, Long> {
-    List<AddressAssessmentDocument> findByOffenderId(Long offenderId);
+    @Query("select document from AddressAssessmentDocument document, AddressAssessment entity where document.addressAssessment = entity and document.offenderId = :offenderId")
+    List<AddressAssessmentDocument> findByOffenderId(@Param("offenderId") Long offenderId);
```

Comment:
Query now limits so parent is not found if child is not found by joining to the table table. Required since Delius has bad data with documents pointing to entities that no longer exist